sites/friday/src/increase-observability/metrics/understand-metrics.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Understand {{< vendor/name >}} metrics
+sidebarTitle: Understand metrics
+description: Understand how to read metrics for {{< vendor/name >}} environments.
+---
+
+{{< vendor/name >}} environments consist of:
+
+* App containers: one or more [app containers](../../create-apps/_index.md)
+* Service containers: zero or more [service containers](../../add-services/_index.md)
+* Worker containers: zero or more [worker instances](../../create-apps/app-reference.md#workers).
+
+Infrastructure metrics report CPU, RAM, and disk space for app and worker containers
+and CPU and disk space for service containers.
+These metrics are available for all of your environments.
+
+App containers are shown first, with the app name and an image corresponding to the app type.
+Service containers follow next with the same pattern and worker containers are shown last.
+
+You can collapse the graphs by clicking **Hide metrics**.
+The graphs switch to an overview of the average resource utilization for the selected container.
+
+![How service container metrics look when minimized](/images/metrics/service-container-minimized.png "0.65")
+
+## Example of how to read metrics
+
+This example should give you an idea of how the metrics appear.
+Environment metrics show resource usage for each app, service, and worker container.
+
+Note that resources are spread across all containers within the project.
+So the resources you see for a given container don't equal the total resources for the project.
+
+This reference project has a single app, two services (MySQL and Redis), and two workers.
+The appropriate resources have been [allocated automatically](../../create-apps/app-reference.md#sizes) for each container
+based on the number and type of containers for this plan size.
```

Comment:
I believe this should be removed as we don't have plans anymore and resources are allocated through container profiles and `resources:set`?